netmiko/base_connection.py

```diff
@@ -1320,6 +1321,20 @@ def find_prompt(
                     self.write_channel(self.RETURN)
                     time.sleep(sleep_time)
                     prompt = self.read_channel().strip()
+
+                    vxr_pattern = "last login"
```

Comment:
i dont think this is needed as vxr has a overrided find_prompt

Reply:
removed